### ISSUE TITLE
SQL execution failed column "starttime" does not exist

### ISSUE THREAD

**Issue by younes-abouelnagah** [PredictionIO/PredictionIO#162]
Trying to use ```LEventStore.find``` specifying startTime and untilTime, but I am getting the following error:

``` [ERROR] [StatementExecutor$$anon$1] SQL execution failed (Reason: ERROR: column "starttime" does not exist  Position: 305):```

Looking at the logged select statement, it does containt startTime and even endTime:

```select id, event, entityType, entityId, targetEntityType, targetEntityId, properties, eventTime, eventTimeZone, tags, prId, creationTime, creationTimeZone from pio_event_1_1 where startTime >= '2014-
10-18 00:00:00.0' and endTime < '2014-10-19 00:00:00.0' and entityType = 'GeoZone' and entityId = '56537' and event = 'open' and targetEntityType = 'Count' order by eventTime asc```

This is not inline with the schema of the events table (luckily I am currently testing out using PostgreSQL) where the only column is ```eventTime```

**Comment by younes-abouelnagah** [comment]
I forgot to mention that I am using the PredictionIO from the AWS as marketplace

**Comment by pferrel** [comment]
What Template?


On Oct 26, 2015, at 8:41 AM, Younes Abouelnagah <notifications@github.com> wrote:

I forgot to mention that I am using the PredictionIO from the AWS as marketplace

**Comment by younes-abouelnagah** [comment]
I started with the vanilla, but changed it a lot by now.

<http://corp.flipp.com>

Younes Abouelnagah
Sr. Data Science Engineer
Flipp Corporation

p: 416-668-3423
e: younes.abouelnagah@flipp.com

On Mon, Oct 26, 2015 at 11:56 AM, Pat Ferrel <notifications@github.com>
wrote:

> What Template?
>
>
> On Oct 26, 2015, at 8:41 AM, Younes Abouelnagah <notifications@github.com>
> wrote:
>
> I forgot to mention that I am using the PredictionIO from the AWS as
> marketplace
>
> —
> Reply to this email directly or view it on GitHub <
> https://github.com/PredictionIO/PredictionIO/issues/162#issuecomment-151178225
> >.
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PredictionIO/PredictionIO/issues/162#issuecomment-151182057>
> .
>

-- 
IMPORTANT NOTICE:  This message, including any attachments (hereinafter 
collectively referred to as "Communication"), is intended only for the addressee(s) 
named above.  This Communication may include information that is 
privileged, confidential and exempt from disclosure under applicable law. 
 If the recipient of this Communication is not the intended recipient, or 
the employee or agent responsible for delivering this Communication to the 
intended recipient, you are notified that any dissemination, distribution 
or copying of this Communication is strictly prohibited.  If you have 
received this Communication in error, please notify the sender immediately 
by phone or email and permanently delete this Communication from your 
computer without making a copy. Thank you.

**Comment by younes-abouelnagah** [comment]
Also, I see that you have fixed that in version 0.9.5 (
https://github.com/PredictionIO/PredictionIO/blob/v0.9.5/data/src/main/scala/io/prediction/data/storage/jdbc/JDBCLEvents.scala#L149)
but the AWS marketplace is still at 0.9.4 with the bug (
https://github.com/PredictionIO/PredictionIO/blob/v0.9.4/data/src/main/scala/io/prediction/data/storage/jdbc/JDBCLEvents.scala#L149
)

<http://corp.flipp.com>

Younes Abouelnagah
Sr. Data Science Engineer
Flipp Corporation

p: 416-668-3423
e: younes.abouelnagah@flipp.com

On Mon, Oct 26, 2015 at 11:57 AM, Younes Abouelnagah <
younes.abouelnagah@flipp.com> wrote:

> I started with the vanilla, but changed it a lot by now.
>
> <http://corp.flipp.com>
>
> Younes Abouelnagah
> Sr. Data Science Engineer
> Flipp Corporation
>
> p: 416-668-3423
> e: younes.abouelnagah@flipp.com
>
> On Mon, Oct 26, 2015 at 11:56 AM, Pat Ferrel <notifications@github.com>
> wrote:
>
>> What Template?
>>
>>
>> On Oct 26, 2015, at 8:41 AM, Younes Abouelnagah <notifications@github.com>
>> wrote:
>>
>> I forgot to mention that I am using the PredictionIO from the AWS as
>> marketplace
>>
>> —
>> Reply to this email directly or view it on GitHub <
>> https://github.com/PredictionIO/PredictionIO/issues/162#issuecomment-151178225
>> >.
>>
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/PredictionIO/PredictionIO/issues/162#issuecomment-151182057>
>> .
>>
>
>

-- 
IMPORTANT NOTICE:  This message, including any attachments (hereinafter 
collectively referred to as "Communication"), is intended only for the addressee(s) 
named above.  This Communication may include information that is 
privileged, confidential and exempt from disclosure under applicable law. 
 If the recipient of this Communication is not the intended recipient, or 
the employee or agent responsible for delivering this Communication to the 
intended recipient, you are notified that any dissemination, distribution 
or copying of this Communication is strictly prohibited.  If you have 
received this Communication in error, please notify the sender immediately 
by phone or email and permanently delete this Communication from your 
computer without making a copy. Thank you.

**Comment by pferrel** [comment]
Yes, we need to update the AWS Marketplace. Not so easy unfortunately a manual process--argh.

**Comment by EmergentOrder** [comment]
opened new issue #190 for us to update AWS, closing this one as fixed in 0.9.5
